lib/transport/selenium-webdriver/appium.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path');
+const {WebDriver} = require('selenium-webdriver');
+const {Executor} = require('selenium-webdriver/http');
+const TransportFactory = require('../factory');
+const SeleniumServer = require('./selenium');
+const AppiumServiceBuilder = require('./service-builders/appium');
+const http = require('selenium-webdriver/http');
+const {isObject} = require('../../utils');
+
+
+module.exports = class AppiumServer extends SeleniumServer {
+  static startServer(settings) {
+    const Options = require('./options.js');
+    const opts = new Options({settings});
+    opts.updateWebdriverPath();
+
+    const appiumService = new AppiumServiceBuilder(settings);
+
+    const outputFile = settings.webdriver.log_file_name || '';
+    appiumService.setOutputFile(outputFile);
+
+    return appiumService;
+  }
+
+  get defaultBrowser() {
+    return null;
+  }
+
+  get ServiceBuilder() {
+    return AppiumServiceBuilder;
+  }
+
+  get defaultServerUrl() {
+    return 'http://127.0.0.1:4723';
+  }
+
+  get defaultPort() {
+    return 4723;
+  }
+
+  get defaultPathPrefix() {
+    return '/wd/hub';
+  }
+
+  createSessionOptions(argv) {
+    // TODO: Re-think the below temp solution
+    if (TransportFactory.usingBrowserstack(this.settings)) {
+      return super.createSessionOptions(argv);
+    }
```

Comment:
How do we figure out from BrowserStack config if the session is for automate or app-automate? Should we just check for `browserName`? If `browserName` exists, create a session with Automate, otherwise with App Automate?

Reply:
Yes we can use `browserName` to figure out this